dpnp/dpnp_iface_mathematical.py

```diff
@@ -1750,8 +1750,8 @@ def ediff1d(ary, to_end=None, to_begin=None):
 -------
 out : dpnp.ndarray
     An array with the rounded values and with the same dimensions as the input.
-    The returned array will have the default floating point data type for the
-    device where `a` is allocated.
+    The returned array will have a floating point data type that input can cast
```

Comment:
I wonder if it is a bug in dpnp implementation. It seems `dpnp.fix` has to be aligned with other rounding functions, like `dpnp.trunc`, where dtype of result array is the same as input.
As for now I don't see a reason why we limit to return only floating dtype arrays.